azure-pipelines-steps.yml

```diff
@@ -0,0 +1,14 @@
+
+steps:
+# using cli
+- script: npm install
+  displayName: 'install'
+- script: npm run build
+  displayName: 'build'
+# test with node 6
+- task: NodeTool@0
```

Comment:
don't you want to choose the version of node first?

also, why build twice? you are explicitly doing `npm run build`, and then `npm test` also builds.